packages/react-composites/tests/browser/chat/app/index.tsx

```diff
@@ -30,8 +32,11 @@ const userId = verifyParamExists(params.userId, 'userId');
 // Optional params
 const useFrLocale = Boolean(params.useFrLocale);
 const customDataModel = params.customDataModel;
+const useFileSharing = Boolean(params.useFileSharing);
+const uploadedFiles = params.uploadedFiles ? JSON.parse(params.uploadedFiles) : [];
 
 // Needed to initialize default icons used by Fluent components.
+initializeFileTypeIcons();
```

Comment:
So... we need to update our documentation to ask Contoso to do this as well? :(

I think we should export our own `initializeIcons` that does both of this then.

Reply:
Yes. I agree, time has come to export our own util function.
Maybe we can go one step further and allow our function to block the loading of composite before icons are loaded. To help with flaky tests.

Reply:
We should be able to remove initializeIcons() when we move away from using fluent CDN font icons also (so then goes back to just `initializeFileTypeIcons()`)